cmd/gentypes/gentypes.go

```diff
@@ -173,6 +173,19 @@ func emitToplevelType(typeName string, descJson json.RawMessage) string {
 		log.Fatal(err)
 	}
 
+	var comment string
+	descDescriptionJson, ok := descMap["description"]
+	if ok {
+		if err := json.Unmarshal(descDescriptionJson, &comment); err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	if len(comment) > 0 {
+		comment = prependCommentPerLine(fmt.Sprintf("%s: %s", typeName, comment))
```

Comment:
Based on the name, I thought this prepended the comment before the type definition, so I was pretty confused at first. Suggest renaming: MakeIntoGoComment, CommentOutEachLine. If you really want to emphasize that you are prepending forward slashes to each line, then maybe call prependSlashesPerLine?

Reply:
Done